eZ/Publish/API/Repository/Tests/ContentTypeServiceTest.php

```diff
@@ -1302,6 +1302,14 @@ public function testUpdateContentTypeDraftStructValues($data)
         $updateStruct = $data['updateStruct'];
         $updatedType = $data['updatedType'];
 
+        // update mainLanguageCode of field definitions
+        foreach ($originalType->fieldDefinitions as $fieldDefinition) {
+            $reflection = new \ReflectionClass($fieldDefinition);
+            $reflectionProperty = $reflection->getProperty('mainLanguageCode');
+            $reflectionProperty->setAccessible(true);
+            $reflectionProperty->setValue($fieldDefinition, $updateStruct->mainLanguageCode);
+        }
```

Comment:
Note: since FieldDefinition inherits `mainLanguageCode`, updated content also has this property updated for nested Field Definitions, which is a read-only property. I couldn't find better way to "adjust" these Field Definitions than by Reflection.
If you have a suggestion how to do it better, I'll appreciate it :)